### ISSUE TITLE
Redis port not be changed when running cluster serving by change redis.conf 

### ISSUE THREAD

**Issue by 1243325667** [intel-analytics/analytics-zoo#3552]
Hey guys,
I used to run redis with default port 6379 ,and recently I need to use other port.But the redis port not be changed after changing the config file of redis (redis.conf) and changing the config file of cluster serving (config.yaml).
Can you give me some suggestions ?

**Comment by glorysdj** [comment]
Hi, what do you mean redis port not be changed. Redis itself can not be started at new port or Cluster Serving not listened to new port?

**Comment by glorysdj** [comment]
yes, this is a bug. we will fix this to allow start redis with conf file.

**Comment by Litchilitchy** [comment]
fixing in #3587

**Comment by Litchilitchy** [comment]
fixed.
